person.rb

```diff
@@ -26,4 +26,8 @@ def of_age?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals << rental
```

Comment:
- [ ] Kindly ensure the person itself is set as a parameter when creating the new rental to achieve the 'has-many' association.